website/client/src/components/inventory/items/index.vue

```diff
@@ -458,12 +465,15 @@ export default {
           }
         });
 
-        itemsArray.sort((a, b) => {
-          if (this.sortBy === 'quantity') {
-            return b.quantity - a.quantity;
-          } // AZ
-          return a.text.localeCompare(b.text);
-        });
+        if (this.sortBy === 'quantity') {
```

Comment:
can we add a comment explaining the reasoning behind the need for the quantity snapshot?

Reply:
If you can add a comment here I'll then merge it , thanks :)